scenariolib/event_search.go

```diff
@@ -48,10 +47,26 @@ func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
 			return nil, errors.New("Parameter custom must be a json object (map[string]interface{}) in a search event.")
 		}
 	}
+
+	if e.Arguments["matchLanguage"] != nil {
+		if matchLanguage, validCast = e.Arguments["matchLanguage"].(bool); !validCast {
+			return nil, errors.New("Parameter matchLanguage must be a type bool in SearchEvent")
+		}
+	} else {
+		matchLanguage = false
+	}
+
```

Comment:
There is no reasons to be afraid of elses, If you'would have to test it you would need a branching anyway, I think it hads more "readability" to say if there is no arguments matchLanguage then it should be false.